styles.css

```diff
@@ -461,6 +448,42 @@ a {
   border: 1px solid #c1c7d0;
 }
 
+/* ----------------------- modal section */
+
+/* .fade:not(.show){
+  opacity: 1;
+}
+.modal{
+  display: block !important;
+} */
+
```

Comment:
- Please, let's also remove these commented lines of code to make our codebase clean and maintainable.

Reply:
done